composer.json

```diff
@@ -30,40 +30,41 @@
     "require": {
         "php": "^5.5.9|~7.0",
         "psr/log": "^1.0",
-        "symfony/config": "^2.7|^3.0",
-        "symfony/debug": "^2.7|^3.0",
-        "symfony/dependency-injection": "^2.7|^3.0",
-        "symfony/event-dispatcher": "^2.7|^3.0",
-        "symfony/finder": "^2.7|^3.0",
-        "symfony/framework-bundle": "^2.7|^3.0",
-        "symfony/http-foundation": "^2.7|^3.0",
-        "symfony/http-kernel": "^2.7|^3.0",
-        "symfony/routing": "^2.7|^3.0",
-        "symfony/security-core": "^2.7|^3.0",
-        "symfony/templating": "^2.7|^3.0",
+        "symfony/config": "^2.7|^3.0|^4.0",
+        "symfony/debug": "^2.7|^3.0|^4.0",
+        "symfony/dependency-injection": "^2.7|^3.0|^4.0",
+        "symfony/event-dispatcher": "^2.7|^3.0|^4.0",
+        "symfony/finder": "^2.7|^3.0|^4.0",
+        "symfony/framework-bundle": "^2.7|^3.0|^4.0",
+        "symfony/http-foundation": "^2.7|^3.0|^4.0",
+        "symfony/http-kernel": "^2.7|^3.0|^4.0",
+        "symfony/routing": "^2.7|^3.0|^4.0",
+        "symfony/security-core": "^2.7|^3.0|^4.0",
+        "symfony/templating": "^2.7|^3.0|^4.0",
         "doctrine/inflector": "^1.0",
         "willdurand/negotiation": "^2.0",
         "willdurand/jsonp-callback-validator": "^1.0"
     },
     "require-dev": {
         "sensio/framework-extra-bundle": "^3.0.13",
-        "symfony/phpunit-bridge": "^3.2",
-        "symfony/asset": "^2.7|^3.0",
-        "symfony/form": "^2.7|^3.0",
-        "symfony/validator": "^2.7|^3.0",
-        "symfony/serializer": "^2.7.11|^3.0.4",
-        "symfony/yaml": "^2.7|^3.0",
-        "symfony/security-bundle": "^2.7|^3.0",
-        "symfony/web-profiler-bundle": "^2.7|^3.0",
-        "symfony/twig-bundle": "^2.7|^3.0",
-        "symfony/browser-kit": "^2.7|^3.0",
-        "symfony/dependency-injection": "^2.7|^3.0",
-        "symfony/expression-language": "~2.7|^3.0",
-        "symfony/css-selector": "^2.7|^3.0",
+        "symfony/phpunit-bridge": "^3.2|^4.0",
+        "symfony/asset": "^2.7|^3.0|^4.0",
+        "symfony/form": "^2.7|^3.0|^4.0",
+        "symfony/validator": "^2.7|^3.0|^4.0",
+        "symfony/serializer": "^2.7.11|^3.0.4|^4.0",
+        "symfony/yaml": "^2.7|^3.0|^4.0",
+        "symfony/security-bundle": "^2.7|^3.0|^4.0",
+        "symfony/web-profiler-bundle": "^2.7|^3.0|^4.0",
+        "symfony/twig-bundle": "^2.7|^3.0|^4.0",
+        "symfony/browser-kit": "^2.7|^3.0|^4.0",
+        "symfony/dependency-injection": "^2.7|^3.0|^4.0",
+        "symfony/expression-language": "~2.7|^3.0|^4.0",
+        "symfony/css-selector": "^2.7|^3.0|^4.0",
         "phpoption/phpoption": "^1.1",
         "jms/serializer-bundle": "^1.0",
         "psr/http-message": "^1.0"
     },
+    "minimum-stability": "dev",
```

Comment:
It's fine for me as long as we don't have problems, just tests are more likely to break with this setting.
Let's keep it this way for now, we'll see soon enough if it causes issues :)